packages/azure-http-specs/specs/azure/resource-manager/common-properties/mockapi.ts

```diff
@@ -126,3 +126,58 @@ Scenarios.Azure_ResourceManager_CommonProperties_ManagedIdentity_updateWithUserA
     },
     kind: "MockApiDefinition",
   });
+
+Scenarios.Azure_ResourceManager_CommonProperties_Error_get = passOnCode(404, {
+  uri: "/subscriptions/:subscriptionId/resourceGroups/:resourceGroup/providers/Azure.ResourceManager.CommonProperties/confidentialResources/:resourceName",
+  method: "get",
+  request: {
+    params: {
+      subscriptionId: SUBSCRIPTION_ID_EXPECTED,
+      resourceGroup: RESOURCE_GROUP_EXPECTED,
+      resourceName: "confidential",
```

Comment:
what does the name mean? do we leverage name to differentiate the cases?

Reply:
> do we leverage name to differentiate the cases?

Yes. We don't want resources across scenarios be of the same name, e.g. `Employee`. It'll be a pain importing them when writing tests.
Here I named it originally for the 403 case. Now that the 403 is replaced with 400 case, it does make little sense here. Open to other names, maybe just `ErrorResource`?

Reply:
since the mock always return build-in error. it's better to rename the op.